.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
 - '6.1'
 env:
   matrix:
-  - TEST_BROWSER_NAME=PhantomJS
```

Comment:
Doesn't work for me. First off, karma is hidden under `node_modules`, so it's not in $PATH. Secondly it still craps out:

```
$ PATH=$PATH:node_modules/karma/bin karma run  --browser firefox
27 02 2017 12:29:44.820:ERROR [runner]: There is no server listening on port 9876
```

Running `npm test` in another terminal still doesn't help:

```
$ PATH=$PATH:node_modules/karma/bin karma run  --browser firefox
No captured browser, open http://localhost:9876/
```

Can we improve on this? We don't want to discourage people from running the test suite.